data_pipeline/kubernetes/kubernetes_pipeline_config.py

```diff
@@ -23,6 +24,27 @@ def convert_dict_to_kubernetes_client_object(
     )
 
 
+def get_env_config_dict_for_env_config_dict_list(
+    env_config_dict_list: Sequence[KubernetesEnvConfigDict]
+) -> Mapping[str, KubernetesEnvConfigDict]:
+    return {
+        env_config_dict['name']: env_config_dict
+        for env_config_dict in env_config_dict_list
+    }
+
+
+def get_merged_env_config_dict_list(
+    env: Optional[Sequence[KubernetesEnvConfigDict]],
+    default_env: Optional[Sequence[KubernetesEnvConfigDict]]
+) -> List[KubernetesEnvConfigDict]:
+    if env and default_env:
```

Comment:
We could avoid this `if` statement by just adding `or []` when calling `get_env_config_dict_for_env_config_dict_list`